src/util/orm.py

```diff
@@ -424,6 +424,28 @@ def custom_module_field_as_manual(env, rollback=True, do_flush=False):
         )
         mock_x_email_fields = [r[0] for r in env.cr.fetchall()]
 
+    temp_ir_model_access_ids = []
+    if version_gte("14.0") and custom_models:
+        # 1.2 Version 14.0 and above requires new access rights for transient models,
+        # and the rights are yet to be added post standard upgrade for custom modules.
+        # To avoid access errors, create some temporary access rights for these models.
+        env.cr.execute(
+            """
+                INSERT INTO ir_model_access (
+                                name, model_id, active, perm_read, perm_write, perm_create, perm_unlink
+                            )
+                     SELECT 'tmp_access', ir_model.id, true, true, true, true, true
```

Comment:
As discussed in DM. I wonder if this would be enough
```suggestion
                     SELECT 'tmp_access', ir_model.id, true, false, false, false, false
```
@KangOl any advice here?

Reply:
Yes, it should be enough. It's also safer.

Reply:
For the TestCrawler to pass, it is enough indeed, I've tested up to v16. But in that case I dropped the `false`s as it's not necessary.